lib/datadog/tracing/contrib/roda/configuration/settings.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require_relative '../../configuration/settings'
+require_relative '../ext'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Roda
+        module Configuration
+          # Custom settings for the Roda integration
+          class Settings < Contrib::Configuration::Settings
+            option :enabled do |o|
+              o.default { env_to_bool(Ext::ENV_ENABLED, true) }
+              o.lazy
+            end
+
+            option :analytics_enabled do |o|
+              o.default { env_to_bool(Ext::ENV_ANALYTICS_ENABLED, false) }
+              o.lazy
+            end
+
+            option :analytics_sample_rate do |o|
+              o.default { env_to_float(Ext::ENV_ANALYTICS_SAMPLE_RATE, 1.0) }
+              o.lazy
+            end
+
+            option :distributed_tracing, default: true
```

Comment:
Can Roda be the root span in a trace? Or would it always be under Rack?

Reply:
I think I answered my own question looking at the example trace you linked in the PR description. Seems like it can be a root span.

Different question, could it NOT be a root span?

Reply:
Based on how Rack can be used with Roda, it certainly seems possible that Rack could be considered a parent of Roda in this case. I tested this locally with this PR and ended up with two separate traces: `rack.request` and another one `roda.request`. Would the expectation that instead of two distinct requests, that there just be one trace?

Reply:
After going over this with @marcotc , the `set_distributed_tracing_context!` that I currently have led to a scenario where if the `rack` integration and `roda` integration gets activated, it led to separate traces.

TODO: I'm going to remove this handling of http headers at the `roda` level and write a note to activate the `rack` integration if distributed tracing is expected.